src/auth-service/config/global/email-templates.js

```diff
@@ -1,6 +1,8 @@
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
 
+const baseUrl = process.env.PLATFORM_STAGING_BASE_URL || process.env.PLATFORM_PRODUCTION_BASE_URL;
```

Comment:
Thanks for this @BenjaminSsempala , 

- Could you share some background notes to this variable declaration? I though `environment` specific variables are handled under `config/environments`. Could you shed more light on this? 
- Ideally, the baseURL is supposed to be determined at runtime of the application --- so we might want to just simply call `constants.PLATFORM_BASE_URL` or `constants.ANALYTICS_BASE_URL`. See **attached screenshot** for visual reference.
- On a side note, we shall also need to be mindful of cyclic dependencies as we go about this.
![Screenshot 2024-04-09 at 07 58 56](https://github.com/airqo-platform/AirQo-api/assets/1590213/c10fc1b6-c761-4ea4-a08a-8b2940be1c54)